tools/configuration-converter.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to convert istio configuration to sail operator configuration.
+# In the end of the execution new yaml file will be created with "sail-ISTIO_CONFIG_YAML" name.
+# Usage: ./configuration-converter.sh ISTIO_CONFIG_YAML_WITH_PATH, example:  ./configuration-converter.sh sample_config.yaml"
+
+if [[ -z $ISTIO_VERSION || -z $ISTIO_NAMESPACE ]]; then
+    echo "ISTIO_VERSION and ISTIO_NAMESPACE should be set as environment variable to execute converter."
+    exit
+fi
+
+WORKDIR=$(dirname "$1")
+ISTIO_CONFIG_FILE=$(basename "$1")
+SAIL_CONFIG_FILE="sail-operator-config.yaml"
+
+cd "$WORKDIR" || exit
+cp "$ISTIO_CONFIG_FILE" "$SAIL_CONFIG_FILE"
+
+function add_mandatory_fields(){
+    sail_api=".apiVersion=\"sailoperator.io/v1alpha1\""
```

Comment:
Sail Operator APIs are now v1

```suggestion
    sail_api=".apiVersion=\"sailoperator.io/v1\""
```

Reply:
Done